[AAP-23637] Hook up run command wizard to toolbar buttons

prat98 opened the pull request:
This PR is for hooking up the run command buttons on the inventory host and groups toolbars to the run command wizard.
It also removes the inventory group page action to run a command because the run command wizard only runs commands against entire inventories. So having a run command page action on a group is not necessary in my opinion. The ability to run a command from the inventory groups page is also not there in the old UI

akus062381 commented:
Hey! I know this isn't specifically what this PR addresses, but since it's right in the same section of the UI, I was wondering whether you could edit the Placeholder text here, to something more like "Select Execution Environment" ? This is in the Execution Environment step of the Run Command Wizard.

![image](https://github.com/ansible/ansible-ui/assets/63685497/08f6ea7c-ad95-4178-93d9-5651a003e042)

akus062381 commented:
When the user gets to the Review step of the Wizard, and clicks the Finish button, the UI doesn't respond. And no error shows in the Network tab. All that happens is the Finish button turns red. Are you able to fix this too? 

![image](https://github.com/ansible/ansible-ui/assets/63685497/59542f65-3224-4f99-b6e7-579aad81c420)

nixocio reviewed (commented):
hey @prat98 is there a related issue?

prat98 commented:
@nixocio yes it's https://issues.redhat.com/browse/AAP-23637

prat98 commented:
> When the user gets to the Review step of the Wizard, and clicks the Finish button, the UI doesn't respond. And no error shows in the Network tab. All that happens is the Finish button turns red. Are you able to fix this too?
> 
> ![image](https://private-user-images.githubusercontent.com/63685497/327561155-ddc146af-90a8-43d5-8621-68ff4de241b6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3Njk5MTgsIm5iZiI6MTcxNDc2OTYxOCwicGF0aCI6Ii82MzY4NTQ5Ny8zMjc1NjExNTUtZGRjMTQ2YWYtOTBhOC00M2Q1LTg2MjEtNjhmZjRkZTI0MWI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTAzVDIwNTMzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI0Yjg5ZDg1OGQ1MWI1ZDgzY2YyZDZjNzBiNGRjZWU1ZDYzYWIwNjVjNTZlMjg1N2Y4NTExZTU0ZGEyZTc3ZjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZGWA6TKaSRJ_Nq9zLjz4JwYB-A7TU2hR14dPF9wUuVA)

Fixed this issue, there was a problem with the way the request was being sent. Also updated the error adapter to pick up this error and show a generic error message in case the server returns an error like this again

prat98 commented:
> Hey! I know this isn't specifically what this PR addresses, but since it's right in the same section of the UI, I was wondering whether you could edit the Placeholder text here, to something more like "Select Execution Environment" ? This is in the Execution Environment step of the Run Command Wizard.
> 
> ![image](https://private-user-images.githubusercontent.com/63685497/327559508-08f6ea7c-ad95-4178-93d9-5651a003e042.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3Njk5MTgsIm5iZiI6MTcxNDc2OTYxOCwicGF0aCI6Ii82MzY4NTQ5Ny8zMjc1NTk1MDgtMDhmNmVhN2MtYWQ5NS00MTc4LTkzZDktNTY1MWEwMDNlMDQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTAzVDIwNTMzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTliYzlkNWI1Yjk0MWQ1ZTFkZDUwMzYyYWM5ZDhiMDdjOWU1Mjk2MjdkOTM0ZjExMTEyOWIzOTZmN2I2Njk5YzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OxikrTueEWdvByiaMUt4JmKosGGMtbS1pVGNeHlV4C8)

Add a placeholder prop and updated the placeholder text to what you suggested

akus062381 reviewed (approved):
This looks great, thanks so much!
